src/core/AnimatedCallFunc.js

```diff
@@ -0,0 +1,20 @@
+import AnimatedNode from './AnimatedNode';
+import { adapt } from './AnimatedBlock';
+
+class AnimatedCallFunc extends AnimatedNode {
+  _value;
```

Comment:
What is the being used for? seems that we only set it to -1 and never read

Reply:
The javascript implementation has not been updated since it is not in use.